app-controllers/controllers

```diff
@@ -0,0 +1,12 @@
+const { fetchTopics } = require("../app-models/models");
+
+exports.getTopics = (request, response, next) => {
+  fetchTopics().then((result) => {
+    response.status(200).send(result);
```

Comment:
your data should be sent back as a property value on an object. For example: 
- {topics: results}